src/ui/handlers/starter-select-ui-handler.ts

```diff
@@ -90,6 +90,16 @@ interface LanguageSetting {
   starterInfoYOffset?: number;
 }
 
+const StarterSelectMode = {
+  NONE: 0,
+  SELECTION: 1,
+  FILTER: 2,
+  PARTY: 3,
+  START: 4,
+};
```

Comment:
```suggestion
const StarterSelectMode = {
  NONE: 0,
  SELECTION: 1,
  FILTER: 2,
  PARTY: 3,
  START: 4,
} as const;
```
Using `AnimFrameTarget` as an example...

Before:

![image](https://github.com/user-attachments/assets/154a96c5-2184-4193-a901-7e994eea807b)

After:

![image](https://github.com/user-attachments/assets/45fa4279-a041-489e-afa5-b1709681e056)

Reply:
ah, thought it was only needed if it was exported, thank you!